armi/utils/tests/test_flags.py

```diff
@@ -69,14 +69,46 @@ class F(Flag):
         f2 = F.from_bytes(array)
         self.assertEqual(f, f2)
 
-    def test_collision(self):
-        """Make sure that we catch value collisions."""
+    def test_collision_extension(self):
+        """Ensure the set of flags cannot be programmatically extended if duplicate created.
+
+        TODO: This test should fail when extend bug is fixed.
```

Comment:
Okay, this is going to be odd, but it's in our docs.

I understand there are a bunch of `TODO`s in ARMI, I get that.

But our rule is that no NEW `TODO` statement can be added to the repo.  If you are trying to add a new TODO, that means either:

* You need to complete your code before your PR, it's not complete yet.
* There is an existing bug in the repo, in which case we need to open a ticket.

The _goal_ of the rule is to improve the code quality of the repo, which was problematic in the past.  Does that make sense?

Thanks!

Reply:
@john-science I've opened a new ticket with this issue noted ([#1469](https://github.com/terrapower/armi/issues/1469)).